src/lib/containers/provenance/ProvenanceGraph.tsx

```diff
@@ -375,6 +375,15 @@ const ProvenanceReactFlow = (props: ProvenanceProps) => {
     onEdgesChangedListener,
   ])
 
+  const onPaneScrollFunction: (event?: React.WheelEvent) => void = useCallback(
+    event => {
```

Comment:
Is there a better way to do this.  I don't want this function to be recreated on each render so I utilized useCallback, but I don't really have a valid dependency array (something that causes re-rendering)